recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -27,30 +34,31 @@ def requirements(self):
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 11)
+            check_min_cppstd(self, 11)
 
     def package_id(self):
         del self.info.settings.compiler
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
+        get(self, **self.conan_data["sources"][self.version],
                   strip_root=True, destination=self._source_subfolder)
 
     def _patch_sources(self):
         cmakelists = os.path.join(
             self._source_subfolder, "CMakeLists.txt")
 
-        tools.replace_in_file(cmakelists,
+        replace_in_file(self, cmakelists,
                               "add_subdirectory(msdfgen)", "")
-        tools.save_append(cmakelists,
-                          "install(TARGETS msdf-atlas-gen-standalone DESTINATION bin)")
+        #tools.save_append(cmakelists,
+                          #"install(TARGETS msdf-atlas-gen-standalone DESTINATION bin)")
 
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
-        self._cmake.definitions["MSDF_ATLAS_GEN_BUILD_STANDALONE"] = True
-        self._cmake.configure()
+        self._cmake.configure(variables={
+            "MSDF_ATLAS_GEN_BUILD_STANDALONE": True
+        })
         return self._cmake
```

Comment:
Replace this with `def generate(self):` containing a CMakeToolchain generator. See the cmake template under docs/ for an example.